.github/prompts/best-practices-dp.prompt.md

```diff
@@ -0,0 +1 @@
+I've recently converted my swagger specification to typespec, can you go through the files of my project and make sure they're following the best practices when doing a data-plane migration?
```

Comment:
in which cases should the user pick this prompt instead of the advanced one?

Reply:
Removed the other prompts, they're not completely ready to be checked in yet.